user_management/src/user/views.py

```diff
@@ -17,9 +17,11 @@ def post(self, request):
             validation_errors = self.signup_infos_validation(json_request)
             if not validation_errors:
                 user = User.objects.create(username=json_request['username'],
-                                    email=json_request['email'],
-                                    password=json_request['password'])
-                refresh_token = JWTManager('refresh').generate_token(user.id)
+                                           email=json_request['email'],
+                                           password=json_request['password'])
+                success, refresh_token, errors = JWTManager('refresh').generate_token(user.id)
+                if success is False:
+                    return JsonResponse(data={'errors': errors}, status=400)
                 return JsonResponse(data={'refresh_token': refresh_token}, status=201)
             else:
                 return JsonResponse(data={'errors': validation_errors}, status=400)
```

Comment:
Do
```
if validation_errors:
    return JsonResponse(data={'errors': validation_errors}, status=400)
...
```

Reply:
I did not mean for you to remove the `else:` but to do:

```py
if validation_errors:
    return JsonResponse(data={'errors': validation_errors}, status=400)
user = User.objects.create(username=json_request['username'],
                                              email=json_request['email'],
                                              password=json_request['password'])
success, refresh_token, errors = JWTManager('refresh').generate_token(user.id)
if success is False:
    return JsonResponse(data={'errors': errors}, status=400)
return JsonResponse(data={'refresh_token': refresh_token}, status=201)
```

To avoid tabulating all those lines